brendan-tuckerman/week7/mon/ajax-movie/css/style.css

```diff
@@ -0,0 +1,37 @@
+body{
+    
+    color: #5B58EE;
+}
+li {
+    list-style: none;
+}
+
+h1{
+    color: #8102f899;
+}
+
+
+.allMoviesList {
+    text-align: center;
+    line-height: 1.4;
+}
+
+#singleMovieDetails{
+    text-align: center;
+    line-height: 1.4;
+}
```

Comment:
you can make this DRY-er by grouping the properties

```
.allMoviesList, #singleMovieDetails{
    text-align: center;
    line-height: 1.4;
}

Reply:
Better yet, `text-align: center` is already being inherited from the parent property `.main_container`

You can add the `line-height: 1.4` to `.main_container` and you might not even need to write the above